content/posts/2024-10-08-18f-year-of-launches.md

```diff
@@ -0,0 +1,187 @@
+---
+title: 18F’s year of launches
+date: 2024-10-08
+authors: 
+- 18F
+excerpt: "Over the past year, 18F has helped amazing partner agencies launch new software products to better serve their users.
+
+Here are six of the launches that 18F has supported this year."
+image: /assets/blog/year-of-launches/header.png
+image_alt: "An image of six rocket ships launching into space"
+# tags:
+# - 18f
+# - agency-work
+# - product-launch
```

Comment:
I got weird build errors locally when I tried to build the site with these tags. 

This would be a helpful TLC crew thing to look into!

1. Do other devs see those same build errors locally when the tags are on? What about in the cloud/preview build?
2. If there's a real bug here with the tags, would be helpful to fix.